easybuild/easyconfigs/f/fastqc/fastqc-0.11.4-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'EB_Java'
```

Comment:
Moving those all in the same directory and consistently applying the fix would be nice, yes (assuming it's the same software). Thanks for looking into this!